src/index.html

```diff
@@ -4,10 +4,291 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="icon" href="./images/favicon.png">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header__top">
+        <a class="header__photo-link" href="tel:123456789"></a>
+        <a class="header__link" href="tel:123456789">+1 234 555-55-55</a>
+        <div id="logo" class="header__logo logo"></div>
+        <a class="header__link" href="tel:123456789">Call to order</a>
+      </div>
+
+      <div class="header__title">
+        <div class="header__hr"></div>
+        <h1 class="header__h1">Miami Condo<br>Kings</h1>
+      </div>
+
+      <div
+        id="toggler"
+        class="nav-toggler header__nav-toggler"
+        onclick="toggle()">
+      </div>
+
+      <nav id="nav" class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link--active">Home</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#about-us" class="nav__link">About us</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gallery" class="nav__link">Gallery</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#consultation" class="nav__link">Consultation</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#testimontials" class="nav__link">Testimontials</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#contacts" class="nav__link">Contacts</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content">
+      <section id="about-us" class="container">
+        <div class="about-us">
+          <h2 class="about-us__h">Miami best<br> Real estate <mark class="about-us__mark">here</mark></h2>
```

Comment:
думай о том чтобы твой код было удобно читать
```suggestion
          <h2 class="about-us__h">
            Miami best<br> Real estate
            <mark class="about-us__mark">here</mark>
          </h2>
```